README.md

```diff
@@ -77,7 +77,7 @@ module: {
 
 ## Options
 
-- __naming__: [bem-naming](https://en.bem.info/toolbox/sdk/bem-naming) overrides
+- __naming__: [bem-naming](https://en.bem.info/toolbox/sdk/bem-naming-entity/) overrides
```

Comment:
Maybe it is more convenient to put a link to naming conventions: https://en.bem.info/toolbox/sdk/bem-naming-entity/#naming-conventions

Reply:
Yes, you're right